packages/react/src/index.js

```diff
@@ -48,100 +48,100 @@ const Card = ({
   </Fragment>
 )
 
-class Microlink extends Component {
-  state = { loading: true }
-
-  componentDidMount () {
-    const { setData, noFetch, url } = this.props
-    const fetch =
-      noFetch || !url ? Promise.resolve({}) : fetchFromApi(this.props)
-
-    fetch.then(({ data }) => {
-      const payload =
-        typeof setData === 'function' ? setData(data) : { ...data, ...setData }
-      this.setData(payload)
-    })
+function parseData (data, props) {
+  const imagesProps = [].concat(props.image)
+  const image = someProp(data, imagesProps)
+  const imageUrl = getUrlPath(image)
+  const { title, description, url, video } = data
+  const { color, background_color: backgroundColor } = image || {}
+  return {
+    color,
+    backgroundColor,
+    title,
+    description,
+    url,
+    video,
+    image: imageUrl
   }
+}
 
-  componentDidUpdate (prevProps) {
-    if (prevProps.data !== this.props.data) this.setData(this.props.data)
-  }
+function Microlink (props) {
```

Comment:
This is personal preference but I like `const ComponentName = props => {...}` + `export default ComponentName` - I see you've defined all components with the `function` keyword @axelhzf.

Is there any benefit from this? 🤔

Reply:
I think this is my personal preference.

There was a time when `const ComponentName = props =>` does not play nice with keeping the ComponentName on the React debugger. But it is working now correctly with `create-react-app`